pkgs/development/python-modules/pylint-flask-sqlalchemy/default.nix

```diff
@@ -0,0 +1,44 @@
+{ buildPythonPackage
+#, fetchgit
+, fetchPypi
+, isPy3k
+, lib
+
+, pytest
+
+  # pythonPackages
+, pylint-plugin-utils
+, flask_sqlalchemy
+}:
+
+buildPythonPackage rec {
+  pname = "pylint_flask_sqlalchemy";
+  version = "0.2.0";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "10dam2dm5kvz2wjbsi7zzrm7m7c8y3y3yf6q76xdhrd4l6mmpplf";
+  };
+
+  propagatedBuildInputs = [
+    pylint-plugin-utils
+    flask_sqlalchemy
+  ];
+  checkInputs = [ pytest ];
+
+  checkPhase = ''
+    pytest tests/test_plugin.py
+  '';
+
+  doCheck = false; # Tests don't seem to work
```

Comment:
Whats the error?

Reply:
This is the output of pytest: https://gist.github.com/NomisIV/aa6adc3c78571428f0ff4178c95bd54e

Reply:
Can you change the commit to reflect that? Something like tests are broken with an upstream link to an issue would be ideal.

Reply:
The project seems to be hosted on a private gitlab server, and I don't think I authenticate in order to leave an issue.

[link to the issue page for the repo](https://gitlab.anybox.cloud/rboyer/pylint_flask_sqlalchemy/-/issues)

Reply:
Also, I tried cloning the repo and running the tests manually, and I got the same results. So the build tools are not the problem.